Forms/index.html

```diff
@@ -9,82 +9,96 @@
 </head>
 
 <body>
-    <img src="res/left-img.png" class="img1">
-    <h1>
-        <font size="15" face="segoe print">Алко.net</font>
-    </h1>
+    <div class="head-container">
+        <h1>
+            <font size="15" face="segoe print">Алко.net</font>
+        </h1>
+        <img src="res/left-img.png" class="img1">
+    </div>
 
     <div class="container">
-        <form action="#" method="get" enctype="multipart/form-data" id="form">
-            <div class="first_collum">
+        <div class="first_column">
+            <div>Текст </div>
+        </div>
+        <form id="form">
```

Comment:
No need to provide id="form" along with the form tag. If the form is _not_ unique on the page, then use something like id="beer_form", id="cats_form", id="stuff_that_is_inside_form"

Reply:
Inderstand